modules/ossm-configuring-the-threescale-wasm-auth-module.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assembly:
+//
+// service_mesh/v2x/ossm-threescale-webassembly-module.adoc
+
+[id="ossm-configuring-the-threescale-wasm-auth-module_{context}"]
+= Configuring the `threescale-wasm-auth` module
+
+As the OpenShift administrator, configure the `threescale-wasm-auth` module to authorize HTTP requests to 3scale API Management through an application binary interface (ABI). The ABI defines the interaction between host and the module, exposing the hosts services, and allows you to use the module to process proxy requests.
+
+== The Service Mesh extension
```

Comment:
needs an ID?

Reply:
I'm not sure. Doesn't an ID usually need a corresponding xref or include? And each new subheading is now automatically appearing in the TOC. Is this necessary or can I exclude them using the [discrete] tag?

Reply:
@abrennan89 I just have this outstanding couple of questions for the moment regarding IDs. It's a quick update and then I'll submit the PR to the squad for more eyes on if you're happy with the updates I've made to your reviews. 

Thank you for all your fantastic review comments.

Reply:
> Doesn't an ID usually need a corresponding xref or include?

No, I believe it's for all headings and titles.
"All titles and section headings must have an anchor ID. The anchor ID must be similar to the title or section heading."
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-titles-and-section-headings

> And each new subheading is now automatically appearing in the TOC. Is this necessary or can I exclude them using the [discrete] tag?

Yes, you should be able to use the [discrete] tag to hide them

Thanks for the updates! 🙂

Reply:
Thanks @abrennan89 :+1: